tardis/montecarlo/montecarlo_numba/single_packet_loop.py

```diff
@@ -86,6 +86,9 @@ def single_packet_loop(
             )
 
         elif interaction_type == InteractionType.LINE:
+
```

Comment:
```suggestion
```
No need of a new line just after if/elif - black isn't strict about extra new lines :smile: